apps/showcase/src/components/showcase/forms/forms-pres.component.ts

```diff
@@ -0,0 +1,158 @@
+import { AsyncPipe, CommonModule, formatDate } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input, ViewEncapsulation } from '@angular/core';
+import { ReactiveFormsModule, UntypedFormControl } from '@angular/forms';
+import { O3rComponent } from '@o3r/core';
+import { Localization } from '@o3r/localization';
+import { CustomFormValidation } from '@o3r/forms';
+import { CopyTextPresComponent, FormsEmergencyContactPresComponent, FormsPersonalInfoPresComponent } from '../../utilities';
+import { EmergencyContact, PersonalInfo } from './contracts';
+import { FormsPresTranslation, translations } from './forms-pres.translation';
+import { dateCustomValidator, formsPresValidatorGlobal } from './forms-pres.validators';
+
+@O3rComponent({ componentType: 'Component' })
+@Component({
+  selector: 'o3r-forms-pres',
+  standalone: true,
+  imports: [
+    AsyncPipe,
+    CommonModule,
+    CopyTextPresComponent,
+    FormsEmergencyContactPresComponent,
+    FormsPersonalInfoPresComponent,
+    ReactiveFormsModule
+  ],
+  templateUrl: './forms-pres.template.html',
+  styleUrl: './forms-pres.style.scss',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FormsPresComponent {
```

Comment:
Maybe you can rename it because it does not fit the Presenter pattern (too much logic)